.github/workflows/build.sh

```diff
@@ -14,8 +14,9 @@ make -j8 install
 ls -R ${top}/install
 
 section "======================================== Verify build"
-# Verify that tester linked with cublas or rocblas as intended.
 ldd test/tester
+
+# Verify that tester linked with cublas or rocblas as intended.
 if [ "${device}" = "gpu_nvidia" ]; then
     ldd test/tester | grep cublas || exit 1
```

Comment:
Pedantic:
Why not storing the result of lld like this:
```
ldd_res=$(ldd test/tester)
```
and after:
```
echo ${ldd_res} | grep -i cublas || exit 1
```

Reply:
What's the advantage of using 2 lines instead of 1 line?

Reply:
I was not clear enough:

```suggestion
ldd_res=$(ldd test/tester)

# Verify that tester linked with cublas or rocblas as intended.
if [ "${device}" = "gpu_nvidia" ]; then
    echo ${ldd_res} | grep -i cublas || exit 1
fi
```
And same for ROCm.

Reply:
Ok, this is clearer, to avoid running `ldd` twice. I figure you mean to change lines 17, 21, and 24. Line 17 would need to become 2 lines to record its output in the log:
```
ldd_res=$(ldd test/tester)
echo "${ldd_res}"
```

Reply:
Oh you are right. I forgot the `echo`.